src/uu/free/src/free.rs

```diff
@@ -43,10 +48,62 @@ struct MemInfo {
     swap_free: u64,
     swap_used: u64,
     reclaimable: u64,
+    low_total: u64,
+    low_used: u64,
+    low_free: u64,
+    high_total: u64,
+    high_used: u64,
+    high_free: u64,
+    commit_limit: u64,
+    committed: u64,
+}
+
+impl MemInfo {
+    fn make_new(&self, o: &MemInfo, min: bool) -> MemInfo {
+        // to use min
+        let clos = |a: u64, b: u64| -> u64 {
```

Comment:
what means clos ?

Reply:
clos is a closure that can switch between selecting the minimum value or the maximum value

Reply:
ok, please rename it for what is doing

Reply:
did you try to simplify it this way?

|a: u64, b: u64| if min { a.min(b) } else { a.max(b) };

Reply:
i also just realized that MaxMem should have the lowest free value